### PR TITLE
Implement Camelcase Filter in nunjucks

### PR DESCRIPTION
**Pull request by yjaeseok**
We cannot use camelCase method name before
So I add Camelcase Filter in nunjucks
After this patch you can use camelcase filter in nunjucks file

e.g. (template/interface_header.njk:Line 35)

```Napi::Value {{member.name | camelcase-}}(const Napi::CallbackInfo& info);```

you can use camelcase filter instead of uppercase

ISSUE=#82

### CODE REVIEW

File: examples/calculator.cc [hunk 1]
```diff
@@ -22,6 +22,10 @@ void Calculator::Print(const std::string& message) {
   printf("%s\n", message.c_str());
 }
 
+void Calculator::CamelCase() {
```
Comment (romandev) [inline examples/calculator.cc]: Could you please rename Equals() method to IsEquals() without adding this new method?
(We wanted to use IsEuqals() but used Equals() due to this camel case problem.)

**Review by romandev — APPROVED** [review]
lgtm with nit

File: examples/calculator.cc (hunk 1 above)
Comment (yjaeseok) [reply examples/calculator.cc]: Okay I will do this

**Comment by yjaeseok** [comment]
@romandev as you mentions, I update PR.

Remove CamelCase function and Update equals function name to isEquals.
Please re-review and approve. Thanks :+1:

**Pull request merged**